tests/documentation/DocumentationStyleBearTest.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from bears.documentation.DocumentationStyleBear import DocumentationStyleBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+def load_testfile(test, filename):
+    filepath = os.path.join(os.path.dirname(__file__), "test_files",
+                            "DocumentationStyleBear", filename)
+    with open(filepath) as fl:
+        return fl.read()
+
+good_file = load_testfile("DocumentationStyleBear", "good_file.py")
+bad_file = load_testfile("DocumentationStyleBear", "bad_file.py")
+
+file_sameline = load_testfile("DocumentationStyleBear", "bad_file2.py")
```

Comment:
imo files are totally okay, doesn't clutter so much when having many files. And when using an IDE, switching to another file is a piece of cake :)